tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -319,6 +325,84 @@ void test_area(const Solution& solution, const Strahlkorper<Fr>& strahlkorper,
   CHECK(irreducible_mass == approx(expected_irreducible_mass));
 }
 
+// Let P = surface_integral_of_scalar(J^i \tilde{s}_i \sqrt(-g))
+// where \tilde{s}_i is the spatial unit one-form to the Strahlkorper,
+// normalized with the flat metric \tilde{s}_i \tilde{s}_j \delta^{ij} = 1,
+// and where euclidean_area_element is passed into surface_integral_of_scalar.
```

Comment:
Define g (spacetime metric?), and perhaps (I don't feel strongly) also define J^i (a flux through the surface?)